<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="ML02W7U" name="SpeedSearch Tests - Firefox 17" tests="3" failures="1" timestamp="24 May 2013 22:43:02 GMT" time="232.264" errors="0">
  <testcase name="test00_Mobile_Login" time="15.742" classname="functionalTests.SpeedSearchTests"/>
  <testcase name="test04_SeTestTCSpeedSearchTickets" time="189.139" classname="functionalTests.SpeedSearchTests">
    <failure type="java.lang.AssertionError" message="timeout">
      <![CDATA[java.lang.AssertionError: timeout
	at org.junit.Assert.fail(Assert.java:88)
	at functionalTests.SpeedSearchTests.test04_SeTestTCSpeedSearchTickets(SpeedSearchTests.java:824)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- test04_SeTestTCSpeedSearchTickets -->
  <testcase name="test50_Mobile_LogOff" time="13.823" classname="functionalTests.SpeedSearchTests"/>
</testsuite> <!-- SpeedSearch Tests - Firefox 17 -->
